python/cudf/cudf/tests/test_transform.py

```diff
@@ -22,14 +22,13 @@ def _generic_function(a):
         (lambda x: x in [1, 2, 3, 4], lambda ser: np.isin(ser, [1, 2, 3, 4])),
     ],
 )
-def test_applymap_python_lambda(dtype, udf, testfunc):
+def test_apply_python_lambda(dtype, udf, testfunc):
```

Comment:
This test file is really short and might be a candidate for removal / integration into another test file.

Reply:
I would be in support of removing this. I'm a little surprised the test for `np.isin` works since this is a numpy API I wouldn't expect to be supported by the cuda target. Will sniff around here too.